### PR TITLE
feat: Add postgres

### PR DESCRIPTION
**Pull request by mgagliardo91**

### CODE REVIEW

File: lib/mutex.js [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+          throw new TableLockedError(this.lockTableName);
+        }
+
+        throw e;
+      }
+    });
+  }
+
+  async useTransaction(callback) {
+    const transaction = await this.queryInterface.sequelize.transaction({ solationLevel: 'SERIALIZABLE' });
```
Comment (branchvincent) [inline lib/mutex.js:83]:
```suggestion
    const transaction = await this.queryInterface.sequelize.transaction({ isolationLevel: 'SERIALIZABLE' });
```

File: .github/workflows/build.yaml [hunk 2]
```diff
@@ -0,0 +1,69 @@
+name: Build
```
Comment (branchvincent) [inline .github/workflows/build.yaml:1]: do we want create a publish workflow to upload to npm?

**Review by branchvincent — APPROVED** [review]
looks 🔥

File: .github/workflows/build.yaml (hunk 2 above)
Comment (mgagliardo91) [reply .github/workflows/build.yaml:1]: yea, I was going to go ahead and get this building and then get that automated as well

**Pull request merged**